Source/Core/DolphinQt2/Debugger/CodeViewWidget.cpp

```diff
@@ -100,6 +100,8 @@ void CodeViewWidget::Update()
   if (Core::GetState() != Core::State::Paused && PowerPC::debug_interface.IsBreakpoint(pc))
     Core::SetState(Core::State::Paused);
 
+  bool darkTheme = qApp->palette().color(QPalette::Base).valueF() < 0.5;
```

Comment:
this can be `const bool` also our naming convention is snake-case, so, `dark_theme`

Reply:
done